patchwork-events-entity/src/main/java/net/patchworkmc/impl/event/entity/EntityEvents.java

```diff
@@ -107,6 +107,10 @@ public static boolean onLivingAttack(LivingEntity entity, DamageSource src, floa
 		return MinecraftForge.EVENT_BUS.post(new LivingAttackEvent(entity, src, damage));
 	}
 
+	public static boolean onPlayerAttack(LivingEntity entity, DamageSource src, float damage) {
```

Comment:
This needs a entry in the patchwork-god-classes module

Reply:
I do have it in ForgeHooks, is that where it's supposed to be, or should it be somewhere else?

Reply:
They changed the relevant method in ForgeHooks (I implemented both originally.) However, you should change the other method in ForgeHooks to call the correct method in EntityEvents, (and remove the now redundant check from there)

Reply:
Should I invert the result like onPlayerAttack or not?

Reply:
The overall result should be the same as if you called it on Forge.

Reply:
The problem is, if I implement it the same as forge, it prevents all entities in the world from attacking...

Reply:
You're not implementing it the same as forge though.

Forge is:
```java
entity instanceof PlayerEntity || !MinecraftForge.EVENT_BUS.post(new LivingAttackEvent(entity, src, damage))
```

while you currently effectively have:
```java
!(entity instanceof PlayerEntity || MinecraftForge.EVENT_BUS.post(new LivingAttackEvent(entity, src, damage)))
```

please note the location of the inversions.

Reply:
I've already tested doing the same thing as forge locally, it just prevents me from attacking and entities from attacking

Reply:
Then please at least commit that behavior, because it is currently definately incorrect.

Reply:
I've updated it, although neither the player nor entities can attack now.

Reply:
Yes, that's because given how this method is used, the logic is now `!(entity instanceof PlayerEntity || !MinecraftForge.EVENT_BUS.post(new LivingAttackEvent(entity, src, damage)))`. So, either change how the method is used (remove the inversion) or invert the entire result.